### PR TITLE
U/womullan/pst cookie

### PR DESCRIPTION
**Pull request by womullan**
fix cookie cutter to allow for ORG not just dm - add PST and PSTN

### CODE REVIEW

**Comment by timj** [comment]
Looks fine but the multiple merges from master need fixing before we merge to master.

File: docs/templates/document.rst [hunk 1]
```diff
@@ -41,9 +41,18 @@ Template configurations
 
 This section describes configurations requested by :command:`cookiecutter`.
 
+``org``
+   Which part of DM the note is from (this comes out on the title page:
```
Comment (jdswinbank) [inline docs/templates/document.rst]:
```suggestion
   Which part of DM the note is from (this comes out on the title page):
```

File: docs/templates/document.rst [hunk 2]
```diff
@@ -41,9 +41,18 @@ Template configurations
...
+   - PST - Project Science Team
+   - DM  - Data Management
+   - SE  - Systems Engineering
+   - PMO - Project Office
+
+
 ``series``
    Handle of the documentation series.
-   Technical notes can be ``DMTN``, ``SQR``, or ``SMTN`` (see the `DM Developer Guide <https://developer.lsst.io/docs/technotes.html>`__ for more information).
+   Technical notes can be ``DMTN``, ``SQR``, ``PSTN'' or ``SMTN`` (see the `DM Developer Guide <https://developer.lsst.io/docs/technotes.html>`__ for more information).
```
Comment (jdswinbank) [inline docs/templates/document.rst]:
```suggestion
   Technical notes can be ``DMTN``, ``SQR``, ``PSTN`` or ``SMTN`` (see the `DM Developer Guide <https://developer.lsst.io/docs/technotes.html>`__ for more information).
```

File: templates/document/cookiecutter.json [hunk 3]
```diff
@@ -1,15 +1,22 @@
 {
+  "org": [
+    "DM",
+    "PST",
+     "SE",
```
Comment (jdswinbank) [inline templates/document/cookiecutter.json]:
```suggestion
    "SE",
```

File: templates/document/cookiecutter.json [hunk 4]
```diff
@@ -1,15 +1,22 @@
...
+     "PMO"],
```
Comment (jdswinbank) [inline templates/document/cookiecutter.json]:
```suggestion
    "PMO"],
```

File: docs/templates/document.rst (hunk 1 above)
Comment (jdswinbank) [reply docs/templates/document.rst]: Actually, most of these aren't “part of DM”. How about:
```suggestion
   Which part of the LSST project the note is from (this is displayed on the title page):
```

**Pull request merged**